Source/EPUBCore/FRBook.swift

```diff
@@ -40,18 +40,18 @@ open class FRBook: NSObject {
     // http://www.idpf.org/epub/301/spec/epub-mediaoverlays.html#sec-package-metadata
 
     var duration: String? {
-        return metadata.findMetaByProperty("media:duration")
+        return metadata.find(byProperty: "media:duration")?.value
     }
 
     var activeClass: String {
-        guard let className = metadata.findMetaByProperty("media:active-class") else {
+        guard let className = metadata.find(byProperty: "media:active-class")?.value else {
```

Comment:
I don't get why use `.value` here and not wrap it in the function as it was before.

Reply:
Since metadata is an object it does not make sense a method named `findMetaByProperty()` or `find(byProperty:)` returns a `String`. In the previous implementation there were two methods named `findMetaByName` and `findMetaByProperty` and they were returning different "contents" that is weird. I guess now these method are more consistent.

Reply:
Yeah makes sense.